### ISSUE TITLE
Status peding_refund

### ISSUE THREAD

**Issue by lucasgomide** [pagarme/pagarme-ruby#21]
When I refund a transaction, using the API, i get the status `paid`. But maybe could be interesting return another status like, `pending_refund`

**Comment by raphaelcuba** [comment]
:+1:

**Comment by dalthon** [comment]
The main problem to adopt this status in this scenario is what pending_refund means.

It means that refund is scheduled to be performed later (due to transfer delay or other tech details).

You may have observed this behaviour when you created a transaction with a postback param, meaning that this transaction is async, and current status after request is still paid, and you will receive a postback when status is changed, maybe transaction will not be refunded (for example, bacause it's already refunded or charged back).

If you don't set a postback (not advised due to timeouts and how yout app will handle it) request will be sync, so status should be refund or maybe pending_refund or whatever status, that actually is the current status of transaction.

In other words, out API returns the actual status of transaction to keep consistency with all statuses really means.

We are currently working on a v2 API that will handle it a little bit differently, and that will avoid these misunderstandings related to concurrency and sync details.

**Comment by lucasgomide** [comment]
Okay. Really the flow above `refund` not is so simple. Because exist some cases that the refund by API don't work, only change status, but not do the refund with emissor.

Ok, we'll waiting for that v2.

I hope a better documantation too. Small detatils that make a big difference.

tks for all
